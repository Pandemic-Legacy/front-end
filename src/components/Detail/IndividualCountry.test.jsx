import React from 'react';
import { shallow } from 'enzyme';
import IndividualCountry from './IndividualCountry';
import { ThemeProvider } from '@material-ui/core';

describe('IndividualCountry component', () => {
  it('renders IndividualCountry', () => {
    const wrapper = shallow(
      <ThemeProvider>
        <IndividualCountry />
      </ThemeProvider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
