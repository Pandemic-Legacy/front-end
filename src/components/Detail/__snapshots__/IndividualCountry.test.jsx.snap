// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndividualCountry component renders IndividualCountry 1`] = `
<ContextProvider>
  <IndividualCountry />
</ContextProvider>
`;
